src/vsr/replica_test.zig

```diff
@@ -2247,7 +2244,8 @@ const TestReplicas = struct {
 
         for (t.replicas.const_slice()) |r| {
             log.info("{}: restart replica", .{r});
-            t.cluster.replica_restart(r, &releases_bundled) catch |err| {
+            t.cluster.replica_set_releases(r, &releases_bundled);
+            t.cluster.replica_restart(r) catch |err| {
```

Comment:
Should we add a TODO to remove `open_upgrade` and replace it with a `set_releases` analogue?

Reply:
I think for replica test `open_upgrade` is fine, as it'll lead to a simpler test, I would think!

We'll need set_release only when we'll write test specifically targeting dynamic behavior.